app/models/runs_extractors.rb

```diff
@@ -47,7 +47,11 @@ def extract(classification)
       end
     end
 
-    raise (StandardError.new('One or more extractors failed').tap{ |e| e.set_backtrace exception.backtrace }) unless exception.blank?
+    exceptions.each do |exception|
+      Rollbar.log('error', exception)
```

Comment:
Could these be reported asynchronously inline when capturing the exception vs storing and then reporting after all extractors are run? https://docs.rollbar.com/docs/ruby#section-asynchronous-reporting

Reply:
yes, that definitely makes more sense. i'll make that change before merging